backend/ecommerce/schema.py

```diff
@@ -143,6 +143,24 @@ def mutate(self, info, name, email, phone, password):
         )
 
 
+class DeleteReview(graphene.Mutation):
+    id = graphene.String()
+
+    class Arguments:
+        productId = graphene.String()
```

Comment:
As we will be having reviewId also  on front-end so we will send that only

Reply:
Since there can be only one review/product why not send product? This way we don't have to check that the person deleting is the owner of the review or not...

Reply:
yeah but it's more intuitive using reviewId and you can easily verify if the review is of that user from review_set